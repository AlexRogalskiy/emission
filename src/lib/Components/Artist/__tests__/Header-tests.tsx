import { NativeModules } from "react-native"

import React from "react"
import * as renderer from "react-test-renderer"

import Header from "../ArtistHeader"

import { Theme } from "@artsy/palette"

beforeAll(() => {
  NativeModules.ARTemporaryAPIModule = { followStatusForArtist: jest.fn() }
})

it("renders properly", () => {
  const artist = {
    internalID: "some-id",
    id: "marcel-duchamp",
    name: "Marcel Duchamp",
    nationality: "French",
    birthday: "11/17/1992",
    counts: {
      follows: 22,
    },
  }
  const header = renderer
    .create(
      <Theme>
        <Header artist={artist as any} />
      </Theme>
    )
    .toJSON()
  expect(header).toMatchSnapshot()
})
