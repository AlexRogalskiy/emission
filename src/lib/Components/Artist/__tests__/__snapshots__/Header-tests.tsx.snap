// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<ArtistHeader
  artist={
    Object {
      "birthday": "11/17/1992",
      "counts": Object {
        "follows": 22,
      },
      "id": "marcel-duchamp",
      "internalID": "some-id",
      "name": "Marcel Duchamp",
      "nationality": "French",
    }
  }
/>
`;
